### PR TITLE
Climate policies - Indicators page

### PR DESCRIPTION
**Pull request by aabdaab**
This PR adds Indicators tab content to climate policies module. It adds two-level accordion table, first grouped by categories and then by titles.

This thread on basecamp explains the data structure for indicators: https://basecamp.com/1756858/projects/15229632/messages/82474964
According to it, there will be changes in the data to provide unique titles for the indicators; for now there are duplicates in the accordion table so we will need to revisit Indicators page once the data is updated and check if the unique value is the `title` as we assume in this PR (this is true for the second level of accordion, the first level is grouped by `category`).

image

### CODE REVIEW

**Comment by kfilip94** [comment]
Hey Marta, few elements are inconsistent with designs, probably first three are not connected with this PR but maybe it'd be good to fix them in the future:


screenshot from 2019-01-09 15-24-17

1. On the left side of `Back to Climate Policies` an arrow is missing `<`
2. Lack of text opacity 75%
3. Selected section should have a text opacity 100%

4. Change font style to uppercase
5. I think the font weight here should be on `bold`, and background should be gray, also spacing and padding should be bigger
6. Grey background color here
7. This border here should be removed

screenshot from 2019-01-09 15-43-07

8. I think you should set maximal width of an element here, because this text can be really long

9. Add `:` at the and of the label

10. Padding here should be bigger
11. Links on the design have a different style

File: app/javascript/app/pages/climate-policies/indicators/indicators-component.jsx [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+
+    return (
+      <div className={styles.page}>
+        <div className={styles.titleContainer}>
+          <div className={styles.title}>
+            Indicators
+          </div>
+        </div>
+        {
+          indicators && indicators && (
```
Comment (kfilip94) [inline app/javascript/app/pages/climate-policies/indicators/indicators-component.jsx]: I believe you can remove the second `indicators` in this condition :)

File: app/javascript/app/pages/climate-policies/indicators/indicators-component.jsx [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+            loading={false}
+            data={indicators}
+            openSlug={openSlug}
+            handleOnClick={this.handleAccordionOnClick}
+            theme={{
+                  title: styles.accordionTitle,
+                  header: styles.accordionHeader
+                }}
+          >
+            {indicators.map(indicator => (
```
Comment (kfilip94) [inline app/javascript/app/pages/climate-policies/indicators/indicators-component.jsx]: You can use destructuring for `indicator` here

**Comment by aabdaab** [comment]
Nice catches, thanks :)
All of the changes are addressed but some of them (the padding ones) are not always exactly like on the design because it looked weird in some cases. I made the paddings a bit smaller.

**Comment by aabdaab** [comment]
Actually wait, I noticed the arrow looks ugly and it's because it should be svg. One sec.

**Comment by aabdaab** [comment]
Ok, ready for review again.

**Comment by kfilip94** [comment]
Thanks for changes :+1: ;)
Please also check background colors, because they are different:
untditled
Also, responsive design for mobiles isn't implemented, should this be done in this PR?

**Comment by simaob** [comment]
Kamil et al, mobile designs are not a priority at this stage, given our tight deadline. Good to do if it doesn't side track people too much from completing the task. We'll try to go through things later to tidy up mobile styles. Unless it's a quick change or tweak that we can do straight away.

**Comment by kfilip94** [comment]
@simaob thanks for an explanation, so I agree it would be better leave it for now :)

**Pull request merged**